malaria24/ona/models.py

```diff
@@ -177,7 +177,7 @@ def get_digest_email_data(self):
                         .last().create_date_time
                     if end_date > max_date:
                             max_date = end_date
-                    province_week = "{0} to {1}".format(min_date.strftime(
```

Comment:
@rudigiesler below is the error I got when I tried to manually trigger a send_digest_email from shell:

Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/var/praekelt/python/local/lib/python2.7/site-packages/celery/local.py", line 167, in <lambda>
    __call__ = lambda x, *a, **kw: x._get_current_object()(*a, **kw)
  File "/var/praekelt/python/local/lib/python2.7/site-packages/celery/app/task.py", line 420, in __call__
    return self.run(*args, **kwargs)
  File "/var/praekelt/malaria24-django/malaria24/ona/tasks.py", line 148, in compile_and_send_digest_email
    NationalDigest.compile_digest().send_digest_email()
  File "/var/praekelt/malaria24-django/malaria24/ona/models.py", line 242, in send_digest_email
    context = self.get_digest_email_data()
  File "/var/praekelt/malaria24-django/malaria24/ona/models.py", line 194, in get_digest_email_data
    'week': province_week,
UnboundLocalError: local variable 'province_week' referenced before assignment


Which made me realize that when I was fixing the dates issue, I renamed the variables (in order to keep track of progress on my side), and those changes are affecting the  overall digest. I changed all the variables back to "week", but I will write an additional test.
QA would had probably gotten the same error had I not tried to manually trigger the celery task